backend/flindt/round/models.py

```diff
@@ -43,19 +43,40 @@ def __str__(self):
 
     def message_for_open(self):
         """
-        TODO: FEED-20: Call this method when a new round is started.
+        Send a message that a new round has started.
```

Comment:
Below there are some comments that I typed before I realized that the entire function can be simplified to this code:

```
...
sending_user_pks = Feedback.objects.filter(round=self.pk).values_list('sender__pk', flat=True).distinct()
for user in sending_user_pks:
    messenger = Messenger(user=sender)
    messenger.send_message(message)
```
This basically means, give me all distinct senders (pks) in this round. If this works, then you can ignore the comments below.

I typed this in my shell_plus and it seemed to do the trick:

Feedback.objects.filter(round=Round.objects.first()).values_list('sender__pk', flat=True).distinct()
```